public/locales/en-US/translations.json

```diff
@@ -207,7 +207,8 @@
   "transaction_empty_title": "No transaction hash supplied",
   "transaction_empty_hint": "Enter a transaction hash in the search box",
   "validator_not_found": "Validator not found",
-  "check_transaction_hash": "Please check your transaction hash",
+  "check_transaction_hash": "Please check your transaction hash or CTID.",
+  "wrong_network": "This CTID applies to a different network.",
```

Comment:
nit: null versions of this translation should be added to the other languages as well